packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -915,7 +934,7 @@ export default class NewVm extends BaseComponent {
                   ) || !this._availableResources()
                 }
                 form='vmCreation'
-                handler={this._create}
+                handler={isAdmin ? this._create : this._selfCreate}
```

Comment:
What if it's a user with ACLs?

Reply:
What do you mean ? we can create a VM either with the resources set or if we are admin.